src/components/NavBar.js

```diff
@@ -1,22 +1,30 @@
 import React from 'react';
 import { NavLink } from 'react-router-dom';
+import '../Styles/Header.css';
+import user from '../Styles/icon.png';
 
 const NavBar = () => {
   const links = [
     { id: 1, path: '/', text: 'books' },
     { id: 2, path: '/categories', text: 'categories' },
   ];
   return (
-    <nav className="navbar">
-      <h1>Bookstore CMS</h1>
-      <ul>
-        {links.map((link) => (
-          <li key={link.id}>
-            <NavLink to={link.path} exact>{link.text}</NavLink>
-          </li>
-        ))}
-      </ul>
-    </nav>
+    <div className="NavBar">
+      <nav className="panel-bg">
+        <h1>Bookstore CMS</h1>
+        <ul>
+          {links.map((link) => (
+            <li key={link.id}>
+              <NavLink to={link.path} exact>{link.text}</NavLink>
+            </li>
+          ))}
+        </ul>
+      </nav>
+      <div className="Oval">
+        <img src={user} alt="user" />
+      </div>
+
+    </div>
```

Comment:
## Optional
- [ ] Please design your navbar like the given [design](https://zeplin.io/)
![Screenshot from 2022-10-08 23-53-01](https://user-images.githubusercontent.com/44745855/194729116-1bbd4a92-fc6e-432e-96ee-9a69fc9adf7b.png)